src/validateInvocation.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  IntegrationExecutionContext,
+  IntegrationProviderAuthenticationError,
+} from '@jupiterone/integration-sdk';
+
+import { createServicesClient } from './collector';
+
+export default async function validateInvocation(
+  context: IntegrationExecutionContext,
+): Promise<void> {
+  try {
+    const client = createServicesClient(context.instance);
+    await client.test();
+  } catch (err) {
+    throw new IntegrationProviderAuthenticationError({
+      cause: err,
+      endpoint: 'api.bugcrowd.com',
+      status: 401,
+      statusText: err.toString(),
+    });
```

Comment:
Great! I think it could be helpful in some integrations to have `endpoint` include the complete URL that was attempted, though I don't know that would be necessary here.